python/tskit/drawing.py

```diff
@@ -403,13 +403,15 @@ def draw_tree(
         warnings.warn(
             "tree_height_scale is deprecated; use time_scale instead",
             FutureWarning,
+            stacklevel=4,
```

Comment:
One of the pre-commit plugins was complaining about its absence. With `stacklevel` the user gets more than one line of the stacktrace with a warning.

Reply:
B028: No explicit stacklevel argument found. The warn method from the warnings module uses a stacklevel of 1 by default. This will only show a stack trace for the line on which the warn method is called. It is therefore recommended to use a stacklevel of 2 or greater to provide more information to the user.